### PR TITLE
Remove ngeo

### PR DESCRIPTION
**Pull request by oterral**
[Test the import WMS/WMTS/GPX/KML](//mf-geoadmin3.int.bgdi.ch/teo_rmngeo/index.html)

### CODE REVIEW

**Comment by oterral** [comment]
jenkins test

File: test/specs/ngeo/modules/import/WmsGetCapDirective.spec.js [hunk 1]
```diff
@@ -1,101 +0,0 @@
...
-      $rootScope = $injector.get('$rootScope');
-      $compile = $injector.get('$compile');
-      $window = $injector.get('$window');
-    });
-
-    map = new ol.Map({});
-    map.setSize([600, 300]);
-  });
-
-  describe('loads a correct WMS GetCapabilities', function() {
```
Comment (loicgasser) [inline test/specs/ngeo/modules/import/WmsGetCapDirective.spec.js]: I think this one is missing in ga tests

File: test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js [hunk 2]
```diff
@@ -1,94 +0,0 @@
...
-      expect(scope.limitations).to.be(undefined);
-      expect(scope.userMsg).to.be(undefined);
-      expect(scope.options.layerSelected).to.be(null);
-      expect(scope.options.layerHovered).to.be(null);
-      expect(scope.addLayerSelected).to.be.a(Function);
-      expect(scope.getAbstract).to.be.a(Function);
-    });
-  });
-
-  describe('loads an invalid WMTS GetCapabilities', function() {
```
Comment (loicgasser) [inline test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js]: this one is missing

File: test/specs/ngeo/modules/import/WmsGetCapDirective.spec.js (hunk 1 above)
Comment (oterral) [reply test/specs/ngeo/modules/import/WmsGetCapDirective.spec.js]: no it exist in https://github.com/geoadmin/mf-geoadmin3/blob/5a0534eeb919d89d8c7fd54a8fd04a752f32f6cd/test/specs/import/WmsGetCapDirective.spec.js#L47

File: test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js (hunk 2 above)
Comment (oterral) [reply test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js]: same here https://github.com/geoadmin/mf-geoadmin3/blob/5a0534eeb919d89d8c7fd54a8fd04a752f32f6cd/test/specs/import/WmtsGetCapDirective.spec.js#L74

Comment (oterral) [reply test/specs/ngeo/modules/import/WmtsGetCapDirective.spec.js]: The test was already duplicated in master

**Pull request merged**